packages/cli/src/controllers/telemetry.controller.ts

```diff
@@ -0,0 +1,59 @@
+import { GlobalConfig } from '@n8n/config';
+import { AuthenticatedRequest } from '@n8n/db';
+import { Get, Post, RestController } from '@n8n/decorators';
+import { NextFunction, Response } from 'express';
+import { createProxyMiddleware, fixRequestBody } from 'http-proxy-middleware';
+
+@RestController('/telemetry')
+export class TelemetryController {
+	proxy;
+
+	constructor(private readonly globalConfig: GlobalConfig) {
+		this.proxy = createProxyMiddleware({
+			target: this.globalConfig.diagnostics.frontendConfig.split(';')[1],
+			changeOrigin: true,
+			pathRewrite: {
+				'^/proxy/': '/', // /proxy/v1/track -> /v1/track
+			},
+			on: {
+				proxyReq: (proxyReq, req) => {
+					proxyReq.removeHeader('cookie');
+					fixRequestBody(proxyReq, req);
+					return;
+				},
+			},
+		});
+	}
+
+	@Post('/proxy/:version/track', { skipAuth: true })
+	async track(req: AuthenticatedRequest, res: Response, next: NextFunction) {
+		await this.proxy(req, res, next);
+	}
+
+	@Post('/proxy/:version/identify', { skipAuth: true })
```

Comment:
Since this endpoints are not protected by authentication, let's add rate limits to them.